tests/clients/test_google-cloud-storage.py

```diff
@@ -10,7 +10,7 @@
 from instana.singletons import tracer
 from ..test_utils import _TraceContextMixin
 
-from mock import patch, Mock
+from mock import patch, Mock, create_autospec
```

Comment:
Are you using this `create_autospec` anywhere?

Reply:
Good catch, this is a leftover from my changes. Will update